talos/manipulate_boxes/script_hpp.py

```diff
@@ -240,9 +240,6 @@
 
 ps.setRandomSeed(123)
 ps.selectPathProjector("Progressive", 0.2)
-# ps.selectPathValidation("Progressive", 0.01)
-ps.selectPathValidation("Discretized", 0.01)
-# ps.selectPathValidation("Dichotomy", 0.0)
```

Comment:
This means we keep the Graph-Discretized path validation method, isn't it ?

Reply:
No  since the path validation method is set in `makeRobotProblemAndViewerFactory`. The above lines used to override the previous choice.

Reply:
Ok.